src/widget/wmainmenubar.cpp

```diff
@@ -126,6 +126,20 @@ void WMainMenuBar::initialize() {
 
     pLibraryMenu->addSeparator();
 
+    QString replaceCueColorTitle = tr("&Replace Cue Colors");
```

Comment:
I'd really prefer to launch this from appropriate context rather than adding to the main menu bar.  My secret plan is to eventually get rid of the menu bar completely -- modern applications don't use them and users have a lot of trouble finding things in them.

Reply:
This is planned, but I need to wait until #2530 has been merged: https://github.com/mixxxdj/mixxx/projects/2#card-34547557

Reply:
that PR is now merged can you update this?

Reply:
I was waiting for #2589 because I didn't want to deal with merge conflicts inside generated Qt Designer files, but I can also do it now if you want.

Reply:
I'll do this in a separate PR after this has been merged.